.github/workflows/msys2_build.yml

```diff
@@ -4,12 +4,6 @@ on:
   # allow manually trigger
   workflow_dispatch:
 
-env:
-  # FIXME: For some reason enabling jit debugging "fixes" random python crashes
-  # see https://github.com/msys2/MINGW-packages/issues/11864 and
-  # https://gitlab.gnome.org/GNOME/glib/-/merge_requests/3280#note_1678973
-  MSYS: winjitdebug
-
```

Comment:
we are disabling JIT, any information in  commit message why ? 

which problems were this caused are they no longer a problem ?